src/test/libradosstriper/aio.cc

```diff
@@ -22,6 +22,7 @@ class AioTestData
   }
 
   ~AioTestData() {
+    sem_unlink("test_libradosstriper_aio_sem");
```

Comment:
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#preparing-and-sending-patches, in this case, "test/librados: " or "ceph_test_rados_api_aio: " would be good enough.